src/main/cpp/bgjs/BGJSV8Engine.cpp

```diff
@@ -121,6 +121,16 @@ static void TraceCallback(const v8::FunctionCallbackInfo<Value>& args) {
     ctx->trace(args);
 }
 
+static void AssertCallback(const v8::FunctionCallbackInfo<Value>& args) {
```

Comment:
We did not do it anywhere in BGJSV8Engine yet, so this basically "fits in" and is okay;
but: I prefer to have the callbacks as static private methods; then we can call a non-static private method from it on the respective instance. => No public methods in the interface, no potential errors.